relays/lib-substrate-relay/src/help.rs

```diff
@@ -0,0 +1,23 @@
+use relay_utils::metrics::{FloatJsonValueMetric, PrometheusError, Registry};
```

Comment:
This file is missing license prelude.

Reply:
Also - when I see a file named `help.rs`, I'm thinking that it is related to help subsystem. WDYT about renaming it to `metrics.rs` or `helpers.rs`?

Reply:
Yes